layers/meta-balena-jetson/conf/machine/photon-tx2-nx.conf

```diff
@@ -0,0 +1,11 @@
+#@TYPE: Machine
+#@NAME: CTI Photon TX2 NX
+##@DESCRIPTION: CTI Photon Carrier Board for NVIDIA Jetson TX2 NX
+#
+
+MACHINEOVERRIDES = "jetson-tx2-nx-devkit:${MACHINE}"
+include conf/machine/jetson-tx2-nx-devkit.conf
+
+# work-around for https://github.com/OE4T/meta-tegra/issues/400
+MACHINEOVERRIDES := "${@'${MACHINEOVERRIDES}'.replace(':${MACHINE}',':jetson-tx2-nx-devkit:${MACHINE}')}"
```

Comment:
Hi @ClintHaerinck 
This override shouldn't be needed anymore. It was fixed in https://github.com/OE4T/meta-tegra/commit/46b020ef63550bd5c6534319824082347e25b58b

Reply:
Thanks, I will commit the change

Reply:
Deleting the 2 lines doesn't work. More investigation is necessary.